milli/tests/search/typo_tolerance.rs

```diff
@@ -170,3 +170,41 @@ fn test_typo_disabled_on_word() {
     let result = search.execute().unwrap();
     assert_eq!(result.documents_ids.len(), 1);
 }
+
+#[test]
+fn test_disable_typo_on_attribute() {
+    let criteria = [Typo];
+    let index = super::setup_search_index_with_criteria(&criteria);
+
+    // basic typo search with default typo settings
+    {
+        let txn = index.read_txn().unwrap();
+
+        let mut search = Search::new(&txn, &index);
+        // typo in `antebel(l)um`
+        search.query("antebelum");
+        search.limit(10);
+        search.authorize_typos(true);
+        search.optional_words(true);
+
+        let result = search.execute().unwrap();
+        assert_eq!(result.documents_ids.len(), 1);
+    }
+
+    let mut txn = index.write_txn().unwrap();
+
+    let config = IndexerConfig::default();
+    let mut builder = Settings::new(&mut txn, &index, &config);
+    // disable typos on `description`
+    builder.set_exact_attributes(vec!["description".to_string()].into_iter().collect());
```

Comment:
You can use the `maplit::hashset` macro and the `big_s::S` function here.

```suggestion
    builder.set_exact_attributes(hashset!{ S("description") });
```